tensorhive/controllers/user/create_user_controller.py

```diff
@@ -36,3 +41,33 @@ def create(user):
         status = 201
     finally:
         return content, status
+
+
+@admin_required
+def create(user):
+    return do_create(user)
+
+
+def create_local(user):
+    # TODO: configure nodes used for authentication
+    auth_node = next(iter(SSH.AVAILABLE_NODES))
```

Comment:
Differentiation of usernames and SSH access of TH between specific hosts is on our list of future features. But it will be closely connected to the changes in task_nursery branch, so for 0.2.4 I decided to provide only this basic functionality.